test/contracts/MainStaking.aes

```diff
@@ -80,6 +80,13 @@ main contract MainStaking =
   entrypoint get_staked_amount(epoch : int) =
     get_staked_amount_(Call.caller, epoch)
 
+  entrypoint get_current_stake() =
+    get_current_stake_(Call.caller)
+
+  entrypoint get_current_stake_(validator : address) =
```

Comment:
function or entrypoint?

Reply:
I think you might want to use it directly in some cases, so it doesn't hurt to have it as an entrypoint, though the name is questionable in that case...